danPortfolio.css

```diff
@@ -0,0 +1,349 @@
+@import url('https://fonts.googleapis.com/css2?family=Itim&display=swap');
+
+@font-face {
+    font-family: "Itim";
+    src: url("../fonts/Itim-Regular.ttf") format('truetype');
+}
+
+* {
+	font-family: 'Itim';
+	box-sizing: border-box;
+	
+}
+
+body {
+	align-self: flex-start;
+    margin: 0;
+}
+
+p, h2, h3, h4 {
+	margin: 0;
+}
+
+p, h1, h2, h3, img {
+	margin-bottom: 20px;
+}
```

Comment:
this is fine except i am wondering why apply 20px bottom margin on all images? seems a bit risky in case you add new images down the line and they have a surprise margin. i would recommend targetting more specifically with class(es) for the ones that need a margin